database/db.js

```diff
@@ -0,0 +1,22 @@
+const pgp = require('pg-promise')()
+
+const cn = {
+  host: 'localhost',
+  port: 5432,
+  database: 'http_auth'
+}
+
+const db = pgp(cn)
+
+const getUser = (email) => {
+  return db.one('SELECT * FROM users WHERE email = $1', [email])
+}
+
+const addUsers = (email, password) => {
```

Comment:
this function name is plural but only adds a single user. Maybe rename it `addUser`?